services/QuillLMS/client/app/bundles/Teacher/components/unitTemplate/__tests__/unitTemplate.test.jsx

```diff
@@ -20,14 +22,71 @@ const unitTemplate = {
   image_link: null
 }
 
-describe('UnitTemplate component', () => {
-  it('should match snapshot', () => {
-    const wrapper = shallow(
+describe('<UnitTemplate />', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('matches the component snapshot', () => {
+    const keeparound = draftjs;
+
+    // jest.mock(draftjs, 'EditorState', () => {
+    //   console.log("MOCKING EDITORSTATE\n\n........")
+    //   return jest.fn()
+    // })
+    // jest.spyOn(draftjs, 'genKey').mockImplementation(() => {
+    //   console.log("MOCKING genKey\n\n........")
+    //   return jest.fn()
+    // });
+
+    jest.spyOn(draftjs, 'ContentState').mockImplementation(() => {
```

Comment:
draftjs creates nondeterministic attribute hashes in snapshots. enzyme ignored this, but react-testing-library does not. Which means that running `jest -u` on this spec file always succeeds, but `jest` always fails on a snapshot mismatch.

According to the jest docs:
https://jestjs.io/docs/snapshot-testing#property-matchers

... and this github issue in draftjs:
https://github.com/facebookarchive/draft-js/issues/702

mocking these two draftjs methods should make make the snapshot generation deterministic. However, my code here is not mocking `EditorState` and `ContentState`. The console logs are not printing.